### PR TITLE
Follow PEP8 for keyword arguments syntax in setup

### PR DESCRIPTION
**Pull request by bittner**
I think it makes sense to offer PEP8 compliant code samples in the docs.

### CODE REVIEW

File: docs/setuptools.txt [hunk 1]
```diff
@@ -130,29 +130,29 @@ dependencies, and perhaps some data files and scripts::
...
-        author_email = "me@example.com",
-        description = "This is an Example Package",
-        license = "PSF",
-        keywords = "hello world example examples",
-        url = "http://example.com/HelloWorld/",   # project home page, if any
+        author="Me",
+        author_email="me@example.com",
+        description="This is an Example Package",
+        license="PSF",
+        keywords="hello world example examples",
```
Comment (bittner) [inline docs/setuptools.txt]: **Side Note:** Must this really be a blank-separated list of word? Or can commas also be used? I've also seen some developers making a list out if this -- Which is the correct way?

**Comment by bittner** [comment]
@jaraco Jason, thanks for merging! Do you know an answer on my question above, too? Is a space-separated string really *the* proposed format? What about commas or even using a list?

**Comment by jaraco** [comment]
In my opinion, the docs should represent what's recommended, the tests should guarantee what's supported, and in lieu of any tests, whatever the implementation supports is what's supported (kinda circular, I know), and we as a community are free to loosen or tighten the spec as we see fit.

**Comment by bittner** [comment]
I agree with you.

- So, the space-separated keywords are the recommended way?
- There doesn't seem to be a test covering the keywords argument.
- I failed to find the implementation of `setup()` where `keywords` is used. Can you help?

**Comment by jaraco** [comment]
> space-separated keywords are the recommended way?

Yes, but that doesn't mean we can't change that if appropriate.

> find the implementation

Look at distutils/dist.py:DistributionMetadata. It looks like the implementation is primarily there. there might be distutils tests that rely on it. It doesn't look like setuptools does much to supplement the behavior in distutils.

**Pull request merged**